src/app/book/store/reducers/book.reducer.ts

```diff
@@ -0,0 +1,14 @@
+import { initialBookState } from '../state/book.state';
+import * as BookPageActions from '../actions/book.actions';
+import { createReducer, on } from '@ngrx/store';
+
+
+export const booksReducers = createReducer (
+  initialBookState,
+  on(BookPageActions.GetBooksSuccess, (state, { books }) => ({ ...state, books })),
```

Comment:
on(
  BookPageActions.GetBooksSuccess, 
  (state, { books }) => ({ ...state, books }),
),

same goes for the whole app, please recheck